packages/yew/src/dom_bundle/bcomp/mod.rs

```diff
@@ -0,0 +1,602 @@
+mod bcomp_impl;
```

Comment:
Why not just put it in this file instead of creating `b*_impl` module? The rest of the project puts code in `mod.rs` so this feels out of place

Reply:
Honestly I wasn't sure, but the impl contains a few things that don't have to be exposed to `lifecycle` and `scope` and isn't so small (200-300 lines is quite a chunk), I think it helps readability a bit, ditto for `btag` with `attributes`/`listeners`. I'm less sure if the tests belong